mobile/ios/Runner/Info.plist

```diff
@@ -47,6 +47,7 @@
       <string>hi</string>
       <string>es</string>
       <string>es</string>
+      <string>es</string>
```

Comment:
You don't need this one since we have `es` above already

Reply:
Thanks! As `es` was already multiple times here, I thought this needed to be 1:1 with the list of locales.
I fixed it, and also removed any existing duplicate language codes.